simulation_parameters/microbe_stage/biomes.json

```diff
@@ -2262,11 +2270,13 @@
               "ConvexShapePath": "res://assets/models/organelles/Chloroplast.shape"
             },
             {
-              "ScenePath": "res://assets/models/organelles/OxytoxyProteins.tscn"
+              "ScenePath": "res://assets/models/organelles/OxytoxyProteins.tscn",
+              "ConvexShapePath": "res://assets/models/organelles/Oxytoxy.shape"
```

Comment:
Or is it used for this now? I think it'd be better to have a different name for the proteins, this is because "oxytoxy" came first so it is the "default" thing that gets to be named oxytoxy and the proteins is the newer thing that should have a different name than just plain oxytoxy.